server/api/api.raml

```diff
@@ -1,1197 +1,1199 @@
-      responses:
-        200:
-          body:
-            application/json:
-              type: string
-        400:
-          description: The input is invalid.
-        500:
-          description: PD server failed to proceed the request.
-    delete:
-      description: Cancel a Region's pending operator.
-      responses:
-        200:
-          description: The pending operator is cancelled.
-        400:
-          description: The input is invalid.
-        500:
-          description: PD server failed to proceed the request.
-
-/hotspot:
-  description: The hot spots status in the cluster.
-  /regions/write:
-    get:
-      description: List the hot write regions.
-      responses:
-        200:
-          body:
-            application/json:
-              type: HotRegions
-  /regions/read:
-    get:
-      description: List the hot read regions.
-      responses:
-        200:
-          body:
-            application/json:
-              type: HotRegions
-  /stores:
-    get:
-      description: List the hot stores.
-      responses:
-        200:
-          body:
-            application/json:
-              type: HotStores
-
-/stats:
-  description: Statistics of the cluster.
-  /region:
-    get:
-      description: Get region statistics of a specified range.
-      queryParameters:
-        start_key?: string
-        end_key?: string
-      responses:
-        200:
-          body:
-            application/json:
-              type: RegionStats
-        500:
-          description: PD server failed to proceed the request.
-
-
-/trend:
-  description: Trend of data growth and movements.
-  get:
-    description: Get the growth and changes of data in the most recent period of time.
-    queryParameters:
-      from: integer
-    responses:
-      200:
-        body:
-          application/json:
-            type: Trend
-      400:
-        description: The request is invalid.
-      500:
-        description: PD server failed to proceed the request.
-
-/admin:
-  /cache/region/{id}:
-    uriParameters:
-      id: integer
-    delete:
-      description: Drop a specific region from cache.
-      responses:
-                200:
-                  description: The region is removed from server cache.
-                400:
-                  description: The input is invalid.
-                500:
-                  description: PD server failed to proceed the request.
-
-  /log:
-    description: The log level of PD server.
-    post:
-      description: Set log level.
-      body:
-        application/json:
-          type: string
-          enum: [ debug, info, warning, error, fatal ]
-      responses:
-        200:
-          description: The log level is updated.
-        400:
-          description: The input is invalid.
-        500:
-          description: PD server failed to proceed the request.
-
-
-/classifier:
-  description: The namespace classifier. Methods depend on current classifier.
+#%RAML 1.0
```

Comment:
Why are all lines changed?

Reply:
I added one more level to all apis under /api/{version}, so almost all lines added one more indent.
However, this won't be a problem any more. I'm going fix the api by the "copy" idea mentioned above, which don't need to modify the docs.